genrl/deep/bandit/dcb_agents/bootstrap_neural.py

```diff
@@ -0,0 +1,130 @@
+from typing import List, Optional
+
+import numpy as np
+import torch
+
+from ..data_bandits import DataBasedBandit
+from .common import NeuralBanditModel, TransitionDB
+from .dcb_agent import DCBAgent
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+dtype = torch.float
```

Comment:
Are these variables necessary?

Reply:
I'll add device as an argument and get rid of dtype